packages/kotti-ui/source/kotti-navbar/KtNavbar.vue

```diff
@@ -101,6 +104,9 @@ export default {
 		toggleMobileMenu() {
 			this.mobileMenuToggle = !this.mobileMenuToggle
 		},
+		toggleNarrow() {
+			this.isNarrow = !this.isNarrow
```

Comment:
`isNarrow` needs to be controlled, especially since yoda is supposed to save those preferences across reloads and since there’s multiple places where the navbar gets rendered.